proxy.conf.json

```diff
@@ -0,0 +1,8 @@
+{
+    "/people": {
```

Comment:
I'm not sure if I'm right, but I think that here in this `proxy.config.json` file you need to add the routes like `/books` , `/quotes`

Reply:
yes! I will update it
havent change from the example people config file